usaspending_api/etl/management/commands/submission_loader.py

```diff
@@ -15,26 +15,37 @@ def add_arguments(self, parser):
         parser.add_argument("--ids", nargs="+", help="list of ids to load", type=int)
 
     def handle(self, *args, **options):
-
         if options["all_time"]:
-            now = datetime.datetime.now()
-            start = datetime.datetime(2017, 1, 1, 0, 0, 0)
-            time_periods = create_full_time_periods(start, now, "q", {})
-            all_periods = [(period["time_period"]["fy"], period["time_period"]["q"]) for period in time_periods]
-            for idx, val in enumerate(all_periods):
-                fy = val[0]
-                q = val[1]
-                try:
-                    logger.info("Running submission load for fy {}, quarter {}...".format(fy, q))
-                    call_command("load_multiple_submissions", int(fy), int(q))
-                except CommandError:
-                    logger.info("Error reported using fy/q combination: {} {}".format(fy, q))
-                    continue
+            self.run_for_all_fiscal_years()
         elif options["ids"]:
```

Comment:
We should probably have it tell the user that at least one of the options must be included if neither are provided

Reply:
Two schools of thought on that, I can go either way..

While the PR is larger than absolutely necessary, I only added logging to the original need to correct the broken behavior which triggered the exception.